tests/integration/sumcoins/main.py

```diff
@@ -1,5 +1,37 @@
 import argparse, redis, json, msgpack
```

Comment:
Was thinking about this earlier, and about what I said.

I think the simplest solution is just allowing every encoding type to define a function which turns the byte slice wallet value into a same typed value such as:

```
{
    "unlocked":{"total": "42", outputs: ["5"]},
    "locked":{"total": "3", outputs: ["1", "2"]},
}

Would keep it all much cleaner, and prevent us from having to define all these keys, and it would also make it that it doesn't matter if the structured value of a certain encoding type has a different schema.